### PR TITLE
Update font used for Affirm and Klarna copy and adjust remove dialog copy

### PR DESCRIPTION
**Pull request by porter-stripe**
## Summary
- Update the copy text color for Affirm and Klarna to match Afterpay.
- Updates the copy on the remove card dialog.

Copy change:
Simulator Screenshot - 15 Pro (new) - 2024-01-04 at 16 26 35

## Motivation
https://jira.corp.stripe.com/browse/MOBILESDK-1358
https://jira.corp.stripe.com/browse/MOBILESDK-1288

## Testing
- Manual

## Changelog
N/A

### CODE REVIEW

File: StripePaymentSheet/StripePaymentSheet/Resources/Localizations/en.lproj/Localizable.strings [hunk 1]
```diff
@@ -1,3 +1,6 @@
+/* Content for alert popup prompting to confirm removing a saved card. {card brand} •••{last 4} e.g. 'Visa •••3155' */
+"%1$@ •••%2$@" = "%1$@ •••%2$@";
```
Comment (yuki-stripe) [inline StripePaymentSheet/StripePaymentSheet/Resources/Localizations/en.lproj/Localizable.strings:2]: Can we double check w/ @jarvis-stripe that this should be 3 dots and not 4?  I think it's usually 4.

File: Example/PaymentSheet Example/PaymentSheetUITest/CustomerSheetUITest.swift [hunk 2]
```diff
@@ -861,7 +861,7 @@ class CustomerSheetUITest: XCTestCase {
     func removeFirstPaymentMethodInList() {
         let removeButton1 = app.buttons["Remove"].firstMatch
         removeButton1.tap()
-        dismissAlertView(alertBody: "Remove Visa ending in 4242", alertTitle: "Remove Card", buttonToTap: "Remove")
+        dismissAlertView(alertBody: "Visa •••4242", alertTitle: "Remove Card?", buttonToTap: "Remove")
```
Comment (yuki-stripe) [inline Example/PaymentSheet Example/PaymentSheetUITest/CustomerSheetUITest.swift:864]: Can we include a screenshot of the change in the PR?

Comment (porter-stripe) [reply Example/PaymentSheet Example/PaymentSheetUITest/CustomerSheetUITest.swift:864]: Added screenshot from latest

File: StripePaymentSheet/StripePaymentSheet/Resources/Localizations/en.lproj/Localizable.strings (hunk 1 above)
Comment (porter-stripe) [reply StripePaymentSheet/StripePaymentSheet/Resources/Localizations/en.lproj/Localizable.strings]: Yeah, I thought that was intentional, but now second guessing. I pinged @jarvis-stripe but for now changed to 4 bullet points to match the rest of our UI.

Comment (porter-stripe) [reply StripePaymentSheet/StripePaymentSheet/Resources/Localizations/en.lproj/Localizable.strings]: Confirmed with @jarvis-stripe. It should be 4 bullets, space, then last 4.

**Review by yuki-stripe — APPROVED** [review]

**Pull request merged**